script/model/training_util/global_data_constructing_util.py

```diff
@@ -244,21 +268,61 @@ def _predict_grouped_opunit_data(data_list, mini_model_map, model_results_path):
 
                 # Poorly encapsulated, but memory scaling factor is located as the 2nd last of feature
                 # slightly inaccurate since ignores load factors for hash tables
-                adj_mem = (pred_mem - buffer_size) * opunit_feature[1][-2] + buffer_size
+                adj_mem = (pred_mem - buffer_size) * opunit_feature[1][-3] + buffer_size
 
                 # Don't modify prediction cache
                 y_pred = copy.deepcopy(y_pred)
                 y_pred[0][data_info.TARGET_CSV_INDEX[Target.MEMORY_B]] = adj_mem
 
             pipeline_y_pred += y_pred[0]
 
+        pipeline_y = copy.deepcopy(pipeline_y_pred)
+        if should_mult and simulate_cache:
+            # Scale elapsed time by 40% (this is a hack)
+            pipeline_y[-1] = pipeline_y[-1] * 0.4
```

Comment:
Though it's a hack, we should also apply to cycles, CPU times, and cache misses.